app/services/rapid_ready_for_decision/fast_track_pdf_generator.rb

```diff
@@ -19,22 +19,20 @@ class FastTrackPdfGenerator
       }
     }.freeze
 
-    def initialize(patient_info, blood_pressure_data, medications)
+    def initialize(patient_info, assessed_data, disability_type)
```

Comment:
Use more-generic `assessed_data`. Plan is to remove data-specific helpers (for things like `:medications`)...